### ISSUE TITLE
Provide apps access to Safe's addressbook / owners list

### ISSUE THREAD

**Issue by TomAFrench** [gnosis/safe-apps-sdk#44]
There are situations when certain apps would benefit from having access to addresses which are relevant to the Safe being used:

- Users can attach real world names to be displayed in dashboards, etc. Useful for viewing Sablier streams or seeing how people voted through governance apps, etc.
- In payments apps users can easily select names from address book in a dropdown to pay repeat clients, reducing errors

This is very much a nice-to-have for when the ecosystem is a bit more developed (plus it has some privacy implications which would need to be addressed)

**Comment by rmeissner** [comment]
> plus it has some privacy implications which would need to be addressed

Very important for me :D this might require something like a permission model/ flow

**Comment by alfetopito** [comment]
On top of that, on my app I'd like to have a list of assets enabled in the current Safe (even if with a balance of `0`), so for example I know what token to display.

**Comment by bh2smith** [comment]
Yes, I am currently writing a “CSV Airdrop” app which could be used by Accounting team to pay out suppliers and even employees. It would be a benefit to them if they were able to cross check the addresses from the CSV they upload with their own address book before submitting the transfers.

https://github.com/bh2smith/safe-airdrop/
